### DISCUSSION TITLE
User experience in winter

### DISCUSSION THREAD

**Discussion by Darellur** [streetcomplete/StreetComplete#2562] [Feedback]
Recently winter disrupted my walks around my area and, besides less comfort with my hand out freezing, some quests, especially those asking for surface materials. Most of ways are covered in snow and I cannot answer them. However, I expect them to be answerable in two months. Hiding quests has two problems:
- I also hide quests with reasons I do not want to disclose and show in the menu just to hide them again
- As to quests undoable for now, hiding surface quests is reachable after a scroll, and three clicks, a bit too much, it takes me less time to just leave the quest

For now I have turned surface quests off and try to not forget to turn them on when the snow melts.

A solution I wouldn't be really satisfied with can be hiding quests for a time, but I'm just asking you all just to consider this situation.

**Comment by mnalis** [comment]
While I do see some use for this, I think allowing users to "snooze" a quest for definable period of time after which it would **automatically reenable itself sounds like a bad idea** (it could create much confusion for users when they forget and quests start turning themselves on and off without user interaction).

It might be a a good candidate for [user-definable quest presets](https://github.com/streetcomplete/StreetComplete/issues/1654#issuecomment-745653725), though - if you could in 2 clicks change from "winter" to "summer" preset, each having only the quests you want and showing current preset name, would that be good enough? 

As a lower-tech solution, you can now just turn off those quests you do not want in winter, and create (in your phone calendar) an event  (dated two months from now) to remind you to turn those StreetComplete quests back on.

**Reply by smichel17** [reply]
> It might be a a good candidate for [user-definable quest presets](https://github.com/streetcomplete/StreetComplete/issues/1654#issuecomment-745653725), though

See also: #2565

**Comment by HolgerJeromin** [comment]
I added it in the list of https://github.com/streetcomplete/StreetComplete/discussions/2457#discussioncomment-352051
